src/Backend/Core/Engine/Meta.php

```diff
@@ -477,6 +477,11 @@ public function save($update = false)
             $custom = null;
         }
 
+		//serialize data for save
+        if(!empty($this->data['data'])){
+            $this->data['data'] = serialize($this->data['data']);
```

Comment:
I would only serialize this when you are inserting or updating it in the database. Now you are changing the data of the object itself.